instances/bodySystem/cardiovascularSystem.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/cardiovascularSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "'Cardiovascular system' is an anatomical system that has as its parts the heart and blood vessels.",
```

Comment:
"'The 'cardiovascular system' is an anatomical organ system where the heart pumps blood through blood vessels to and from all part of the body."